packages/esm-patient-chart-app/src/patient-chart/action-menu/action-menu.component.tsx

```diff
@@ -49,13 +50,7 @@ export const ActionMenu: React.FC<ActionMenuInterface> = ({ open }) => {
       </Button>
     </aside>
   ) : (
-    <Button className={styles.actionBtn}>
-      <div>
-        <Edit20 />
-        {windowSize.size === 'hidden' && <WarningFilled16 className={styles.warningButton} />}
-      </div>
-      <span>{t('careActivities', 'Care Activities')}</span>
-    </Button>
+    <ExtensionSlot state={state} className={styles.extensionStyles} extensionSlotName={'action-menu-items-slot'} />
```

Comment:
The fact that you're passing state in tablet mode but not in desktop mode is a red flag to me. But here, we don't need the state at all! Just use the `useWorkspaces` hook directly in components that need the list of workspaces.

Again: avoid state when possible.